pkg/osquery/tables/execparsers/repcli/test-data/repcli_darwin.txt

```diff
@@ -0,0 +1,77 @@
+General Info: 
+    Sensor Version: 3.7.2.81
+    Kernel Type: System Extension
+    System Extension: Running
+    Kernel File Filter: Connected
+    Background Scan: Complete
+    Sensor Restarts: 1911
+    Last Reset: not set
+    FIPS Mode: Disabled
+Full Disk Access Configurations: 
+    Repmgr: Not Configured
+    System Extension: Unknown
+    OSQuery: Unknown
+    Uninstall Helper: Unknown
+    Uninstall UI: Unknown
+Sensor State: 
+    State: Enabled
+    Details: 
+        Shutdown In Progress
+        LiveResponse:NoSession
+        LiveResponse:NoKillSwitch
+        LiveResponse:Enabled
+        FullDiskAccess:NotEnabled
+    SvcStable: Yes
+    Boot Count: 103
+    First Boot After OS Upgrade: No
+    Service Uptime: 155110500 ms
+    Service Waketime: 37860000 ms
+Cloud Status: 
+    Server Address: https://dev-prod05.example.com
+    Registered: Yes
+    Next Check-In: Now
+    Private Logging: Disabled
+    Next Cloud Upgrade: None
+    MDM Device ID: 9FCF04E4-4C8C-45A0-B3EA-053672776382
```

Comment:
Is this redacted? I'd start it with `99999999` and it'll be obvious

Reply:
this was just from me running `uuidgen` locally, I will update with 9s for clarity thank you!